static/docs/get-started/data-artifacts.md

```diff
@@ -0,0 +1,45 @@
+# Reusing Data Artifacts
```

Comment:
let's skip artifacts .. how about just `reusing data` (I still don't like the name, haha :) )

Reply:
Sure, changed. Maybe **Reusing data among different projects" ?

Reply:
nah, let's keep just reusing data for now .. will figure out something else later if needed